src/jit/hwintrinsiccodegenxarch.cpp

```diff
@@ -1388,6 +1388,29 @@ void CodeGen::genSSEIntrinsic(GenTreeHWIntrinsic* node)
             break;
         }
 
+        case NI_SSE_SetAllVector128:
+        {
+            assert(baseType == TYP_FLOAT);
+            assert(op2 == nullptr);
+            if (compiler->compSupports(InstructionSet_AVX2))
+            {
+                emit->emitIns_R_R(INS_vbroadcastss, emitTypeSize(TYP_SIMD16), targetReg, op1Reg);
+            }
+            else if (compiler->compSupports(InstructionSet_AVX))
+            {
+                emit->emitIns_SIMD_R_R_I(INS_vpermilps, emitTypeSize(TYP_SIMD16), targetReg, op1Reg, 0);
+            }
+            else
+            {
+                if (targetReg != op1Reg)
+                {
+                    emit->emitIns_R_R(INS_movaps, emitTypeSize(TYP_SIMD16), targetReg, op1Reg);
+                }
+                emit->emitIns_R_R_I(INS_shufps, emitTypeSize(TYP_SIMD16), targetReg, targetReg, 0);
```

Comment:
Why not `SIMD_R_R_I`, which should handle the `movaps`?

Reply:
Will fix.